internal/hcsoci/layers.go

```diff
@@ -16,6 +16,29 @@ import (
 	"github.com/pkg/errors"
 )
 
+// AllLayers contains all the layers for an image.
+type AllLayers struct {
```

Comment:
could we call this `ImageLayers` instead?

Reply:
I like this more, Will change